websites/models.py

```diff
@@ -110,7 +110,13 @@ class WebsiteContent(TimestampedModel, SafeDeleteModel):
 
     def upload_file_to(self, filename):
         """Return the appropriate filepath for an upload"""
-        return f"{self.website.name}/{self.text_id.replace('-', '')}_{filename}"
+        site_config = SiteConfig(self.website.starter.config)
+        url_parts = [
+            site_config.root_url_path,
```

Comment:
I tried this setting `root-url-path` to "/" and it triggered an error:

```
nginx_1   | 2021/06/30 18:16:12 [warn] 6#6: *72 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000006, client: 172.18.0.1, server: , request: "POST /api/websites/ocw-www-github/content/ HTTP/1.1", host: "localhost:8043", referrer: "http://localhost:8043/sites/ocw-www-github/type/resource/"
web_1     | [2021-06-30 18:16:12] ERROR 16 [django.security.SuspiciousFileOperation] exception.py:88 - [c3eda0c6a826] - Detected path traversal attempt in '//ocw-www-github/f37b72b4b9714a8793139ddb8eb98d49_bob.png'
web_1     | [2021-06-30 18:16:12] ERROR 16 [django.security.SuspiciousFileOperation] exception.py:88 - [c3eda0c6a826] - Detected path traversal attempt in '//ocw-www-github/f37b72b4b9714a8793139ddb8eb98d49_bob.png'
web_1     | [2021-06-30 18:16:12] WARNING 16 [django.request] log.py:224 - [c3eda0c6a826] - Bad Request: /api/websites/ocw-www-github/content/
web_1     | [2021-06-30 18:16:12] WARNING 16 [django.request] log.py:224 - [c3eda0c6a826] - Bad Request: /api/websites/ocw-www-github/content/
```

We should handle the case of a leading slash being in the path